### PR TITLE
intelmetool: init at 4.8.1

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/intelmetool/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    make -C util/intelmetool
+    '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp util/intelmetool/intelmetool $out/bin
+    '';
+
+  meta = with stdenv.lib; {
+    description = "Dump interesting things about Management Engine even if hidden `C`";
```
Comment (srhb) [inline pkgs/tools/misc/intelmetool/default.nix]: I think we can drop the `C` -- it doesn't make much sense in this context.

**Review by srhb — CHANGES_REQUESTED** [review]
I am wondering whether it would make sense to reuse the src and version from nvramtool, but I guess there's not much precedent for that. The duplication just seems a little off to me, but probably no biggie.

**Comment by srhb** [comment]
@GrahamcOfBorg build intelmetool

**Comment by gnidorah** [comment]
@srhb Also `cbfstool` uses the same repo. Should be tested after version bump though. What would be correct way to share src and version between them? Is there any example in nixpkgs?

**Comment by srhb** [comment]
@gnidorah I haven't found any, so at least it's not prevalent :/

**Comment by srhb** [comment]
@GrahamcOfBorg build intelmetool

**Review by srhb — APPROVED** [review]

**Comment by gnidorah** [comment]
@srhb Thanks!

**Pull request merged**